content/blog/2023/2023-08-31-a-dash-of-dask.md

```diff
@@ -0,0 +1,45 @@
++++
+author = "Noah Novšak"
+date = "2023-08-31"
+draft = false
+title = "A dash of Dask"
+type = "blog"
+thumbImage = ""
+frontPageImage = ""
+blog = ["dask"]
```

Comment:
Maybe we could add a special tag for blogs where we address programers. Any idea how you'd call it?